### PR TITLE
Fox Lab Time80 Layout Macro Additions

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

- add eight new layout macros
- enable Community Layout support
- add layout/matrix diagram 

cc @lukelex (keyboard maintainer)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/foxlab/time80/info.json [hunk 1]
```diff
@@ -21,113 +21,971 @@
...
+                {"label": "Fn3", "matrix": [5, 9], "x": 11.25, "y": 5.25, "w": 1.25},
+                {"label": "Fn4", "matrix": [5, 10], "x": 12.5, "y": 5.25, "w": 1.25},
+                {"label": "Fn4", "matrix": [5, 11], "x": 13.75, "y": 5.25, "w": 1.25},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 12], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 7], "x": 17.25, "y": 5.25}
+            ]
+        },
+
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:143]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 2]
```diff
@@ -21,113 +21,971 @@
...
+                {"label": "GUI", "matrix": [5, 9], "x": 11.25, "y": 5.25, "w": 1.25},
+                {"label": "Fn", "matrix": [5, 10], "x": 12.5, "y": 5.25, "w": 1.25},
+                {"label": "Ctrl", "matrix": [5, 11], "x": 13.75, "y": 5.25, "w": 1.25},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 12], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 7], "x": 17.25, "y": 5.25}
+            ]
+        },
+
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:248]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 3]
```diff
@@ -21,113 +21,971 @@
...
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:355]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 4]
```diff
@@ -21,113 +21,971 @@
...
+                {"label": "Alt", "matrix": [5, 9], "x": 11, "y": 5.25, "w": 1.5},
+                {"label": "Fn", "matrix": [5, 10], "x": 12.5, "y": 5.25},
+                {"label": "Ctrl", "matrix": [5, 11], "x": 13.5, "y": 5.25, "w": 1.5},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 12], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 7], "x": 17.25, "y": 5.25}
+            ]
+        },
+
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:459]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 5]
```diff
@@ -21,113 +21,971 @@
...
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:565]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 6]
```diff
@@ -21,113 +21,971 @@
...
+                {"label": "GUI", "matrix": [5, 9], "x": 11.25, "y": 5.25, "w": 1.25},
+                {"label": "Fn", "matrix": [5, 10], "x": 12.5, "y": 5.25, "w": 1.25},
+                {"label": "Ctrl", "matrix": [5, 11], "x": 13.75, "y": 5.25, "w": 1.25},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 12], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 7], "x": 17.25, "y": 5.25}
+            ]
+        },
+
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:671]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 7]
```diff
@@ -21,113 +21,971 @@
...
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:779]:
```suggestion
```

File: keyboards/foxlab/time80/info.json [hunk 8]
```diff
@@ -21,113 +21,971 @@
...
+                {"label": "Alt", "matrix": [5, 9], "x": 11, "y": 5.25, "w": 1.5},
+                {"label": "Fn", "matrix": [5, 10], "x": 12.5, "y": 5.25},
+                {"label": "Ctrl", "matrix": [5, 11], "x": 13.5, "y": 5.25, "w": 1.5},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 12], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 7], "x": 17.25, "y": 5.25}
+            ]
+        },
+
```
Comment (fauxpark) [inline keyboards/foxlab/time80/info.json:884]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/time80/info.json (hunk 1 above)
Comment (noroadsleft) [reply keyboards/foxlab/time80/info.json:143]: oops

**Review by fauxpark — APPROVED** [review]

**Pull request merged**